lib/util/git/index.ts

```diff
@@ -131,15 +182,18 @@ async function fetchBranchCommits(): Promise<void> {
     );
   }
   try {
-    (await git.raw(opts))
+    (await gitRetry(() => git.raw(opts)))
```

Comment:
Is this really an online function?

Reply:
Hi @viceice I guess it is online: https://git-scm.com/docs/git-ls-remote.html

"Displays references available in a remote repository along with the associated commit IDs."

Since it is related to "remote repository", I think it has to be online.